packages/block-library/src/image/block.json

```diff
@@ -33,6 +33,13 @@
 			"selector": "figcaption",
 			"__experimentalRole": "content"
 		},
+		"lightbox": {
```

Comment:
So by default all images don't have light boxes right?

Reply:
Yup, the lightbox is disabled by default